src/AppBundle/Controller/PagesController.php

```diff
@@ -61,20 +62,15 @@ public function indexAction($project = null)
 
         // set default wiki so we can populate the namespace selector
         if (!$project) {
-            $project = $this->container->getParameter('default_project');
+            $project = $this->getParameter('default_project');
```

Comment:
Could also do `$project = ProjectRepository::getDefaultProject($this->container);`